database/seeds/DatabaseSeeder.php

```diff
@@ -14,12 +14,17 @@ class DatabaseSeeder extends Seeder
     public function run()
     {
         Eloquent::unguard();
-        DB::statement('SET FOREIGN_KEY_CHECKS = 0');
+
+        if (! app()->environment('testing')) {
+            DB::statement('SET FOREIGN_KEY_CHECKS = 0');
+        }
 
         $this->call('UsersSeeder');
         $this->call('TalksSeeder');
         $this->call('ConferencesSeeder');
 
-        DB::statement('SET FOREIGN_KEY_CHECKS = 1');
+        if (! app()->environment('testing')) {
+            DB::statement('SET FOREIGN_KEY_CHECKS = 1');
+        }
```

Comment:
Glad you asked. I don't think that's a final
Solution but rather a hack. But it was causing SQLite errors for the tests.